terraform/environments/panda-cyber-appsec-lab/platform_locals.tf

```diff
@@ -1,6 +1,6 @@
 locals {
 
-  application_name = "panda-cyber-appsec-lab"
+  application_name = "$application_name"
```

Comment:
This will need updating to reflect the customer account name

Reply:
Amended and pushed up